src/client/pages/OfferNew/Introduction/Sidebar/StartDate.tsx

```diff
@@ -210,37 +247,52 @@ const DateForm: React.FC<{
     }
   }
 
+  const setDate = async (newDateValue: Date | null) => {
+    setDateValue(newDateValue)
+    setShowError(false)
+    setIsLoadingPickedStartDate(true)
+    if (newDateValue === null) {
+      try {
+        isSingleBundleStartDate && offerData
```

Comment:
I was a bit confused by the name - what do you think about `isSingleStartDateBundle` instead? Then it's like is it a Bundle of type SingleStartDate